src/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { Task } from "./types";
+
+type Props = {
+  tasks: Task[];
+}
+
+const getIncompleteTasksCount = ({tasks}: Props) => {
```

Comment:
для обычных функций необязательно так делать. Обычно я использую правило, если больше трёх (или даже двух) аргументов использовать объект. В данном случае для одного аргумента это не очень оправданно

можно так
```suggestion
const getIncompleteTasksCount = (tasks: Task[]) => {
```